pitch.md

```diff
@@ -1,60 +1,224 @@
 
+[celtx](https://www.celtx.com/features.html) is a tool to create digital video
+production scripts that generates various scene breakdowns and statistics from
+a script. Since it's aimed at video production it does not include a way to
+express cues. For the part of the project where I will need to digitally represent
+a script, some ideas can be used from this software.
+
+There are also a few threads floating around the internet ([SMNetwork](http://smnetwork.org/forum/tools-of-the-trade/digital-prompt-books-%28do-you-use-them%29/), [Control Booth](https://www.controlbooth.com/threads/electronic-prompt-book.23827/), and
+[an anecdote](https://prezi.com/jquqjnga7lri/creating-a-digital-prompt-book/))
+that discuss many people's aprehension and frustrations with current solutions (which
+are mostly PDF hacks). Many are afraid to adopt a digiital solution since the
+prompt book is so vital and having a computer freeze or malfunction during an
+actual show would be extremely troubling.  
+
+For the PDF solutions people have attempted to create, I think it gives an example
+of an ultimate goal: create a solution that could produce a PDF of the script with
+the appropriate cues giving people the option of printing the script. Alternativly,
+an interactive version could be produced for those interested in putting more
+faith in technology.
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+For this project, there exists a 50/50 split: designing both the cue langaguage
+and a way to encode the script in a form that the cues could be applied to will
+take half the time, while actually implementing it will take the other 50. Adding
+a GUI version, would shift this to 15/85; however, I don't think at this point
+making a GUI would make sense&mdash;unless splitting the project with a partner.
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+Creating a text based interface and the supporting language for this project
+is a nice balance of goals. In a way, in order to write the DSL for cueing,
+I need to make a DSL (or at least an AST) for script to which the cue DSL
+implementation could work with. If I aim to make a GUI, I think the projects
+scope increases and will push on the amount of time to complete it. The GUI I
+imagine would render a digital version of the image of a handwritten prompt
+book.
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+Designing the exact syntax will be tricky. I plan on interviewing some of my
```

Comment:
Sounds like an interesting challenge.  Do you think there will be a way to make it extensible so that users can choose their own syntax, or would too many options be overwhelming?